apps/system/utils/Kconfig

```diff
@@ -81,6 +81,14 @@ config ENABLE_HEAPINFO
 	---help---
 		Show information about memory status per thread
 
+config ENABLE_IRQINFO
+	bool "irqinfo"
+	default y
```

Comment:
@sunghan-chang , I believe all System debug engineers would use this to identify spurious interrupts in real target and to know the cpu overloading statistics. So i left it to be "y"  
Other than this comment, I have fixed rest of your valuable review comments.  Kindly review
https://github.com/Samsung/TizenRT/pull/263/commits/1d01e99fcca438678c0f45e90ce87d51687e7fcd